src/WordPress/JsonMapper/JsonMapper.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace WordPress\JsonMapper;
+
+use WordPress\JsonMapper\Evaluators\DocBlockAnnotations;
+use WordPress\JsonMapper\Evaluators\NamespaceResolver;
+use WordPress\JsonMapper\Evaluators\PropertyMapper;
+use WordPress\JsonMapper\Property\PropertyMap;
+
+class JsonMapper {
+	private $evaluators;
+
+	public function __construct() {}
+
+	private function configure_evaluators() {
+		if ( null === $this->evaluators ) {
+			$this->evaluators = array(
+				new DocBlockAnnotations(),
+				new NamespaceResolver(),
+				new PropertyMapper( $this ), // PropertyMapper has to be the last evaluator.
+			);
+		}
+	}
+
+	public function map_to_class( \stdClass $json, string $class ) {
```

Comment:
I believe this is a generic term:

```suggestion
	public function hydrate( \stdClass $json, string $class ) {
```

Reply:
Done.